### ISSUE TITLE
Add content item selector to the Create Issue/Issue Details dialog

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp#4654]
Depends on #4681 

* When user creates a new issue, he should be able to add content items to it right away.
* When user opens an existing issue, he should be able to add or remove items in the issue

Add a content selector to the dialog, above the button toolbar. What we need is something like LoaderComboBox in the Move dialog, but with multi-select option (checkboxes next to content items).

Selected items will be shown under the textbox with "X" icons to remove from selection.

* For a new issue, we should pre-select items that are already selected in the grid (if any)
* For an existing issue, we should hide the selector and show only the list of items saved with the issue. 
* When the dialog is switched to "edit" mode, then both selector and the selection should be shown.

![image](https://cloud.githubusercontent.com/assets/11612490/24254899/51679c96-0fe4-11e7-910c-633c4efe4bac.png)
